.github/workflows/test.yml

```diff
@@ -81,3 +81,12 @@ jobs:
       run: go test -gcflags=all=-d=checkptr -count 10 -v ./...
       env:
         GO_OPENSSL_VERSION_OVERRIDE: ${{ matrix.openssl-version }}
+  azurelinux:
+    runs-on: ubuntu-latest
```

Comment:
So we're running an Azure Linux container on a ubuntu host?

Reply:
Yep. this is a requirement as per GitHub Actions docs: https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#jobsjob_idcontainer

> Note: If your workflows use Docker container actions, job containers, or service containers, then you must use a Linux runner:

> - If you are using GitHub-hosted runners, you must use an Ubuntu runner